Services/Controllers/UserController.cs

```diff
@@ -0,0 +1,87 @@
+﻿//
+// Community Forums
+// Copyright (c) 2013-2021
+// by DNN Community
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated 
+// documentation files (the "Software"), to deal in the Software without restriction, including without limitation 
+// the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and 
+// to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all copies or substantial portions 
+// of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED 
+// TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL 
+// THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF 
+// CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER 
+// DEALINGS IN THE SOFTWARE.
+//
+using System;
+using System.Data;
+using System.Net;
+using System.Net.Http;
+using System.Reflection;
+using System.Web.Http;
+using DotNetNuke.Entities.Portals;
+using DotNetNuke.Entities.Users;
+using DotNetNuke.UI.UserControls;
+using DotNetNuke.Web.Api;
+using static DotNetNuke.Modules.ActiveForums.Handlers.HandlerBase;
+
+namespace DotNetNuke.Modules.ActiveForums.Services.Controllers
+{
+    /// <summary>
+    /// <inheritdoc/>
+    /// </summary>
+    public class UserController : ControllerBase<UserController>
+    {
+        /// <summary>
+        /// Fired by UI while user is online to update user's profile with 
+        /// </summary>
+        /// <returns></returns>
+        /// <remarks>https://dnndev.me/API/ActiveForums/User/UpdateUserIsOnline</remarks>
+        [HttpGet]
```

Comment:
Since this is an editing action, shouldn't we be using a `POST` instead? 🤔

Reply:
Technically probably yes. The services framework handles passing the UserInfo object back to the API, so there is no data passed back. So I just made it a GET. I can change to a POST if you think that's more appropriate. Let me know...

Reply:
I went ahead and changed it to a `POST`.